GoogleSignIn/Tests/Unit/GIDFakeMainBundle.h

```diff
@@ -22,12 +22,31 @@
  */
 @interface GIDFakeMainBundle : NSObject
 
+/**
+ * @fn initWithClientID:serverClientID:hostedDomain:openIDRealm:
+ * @brief Initializes a GIDFakeMainBundle object.
+ * @param clientID The fake client idenfitier for the app.
+ * @param serverClientID The fake server client idenfitier for the app.
+ * @param hostedDomain The fake hosted domain for the app.
+ * @param openIDRealm The fake OpenID realm for the app.
+ */
+- (nullable instancetype)initWithClientID:(nullable id)clientID
```

Comment:
I think we want this to be the designated initializer. Then, we can implement `-init` to call thru to this, which will pass `nil` to all of the params.

Reply:
Done.